contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2359,14 +2359,14 @@ CREATE OR REPLACE VIEW sys.syslanguages
 AS
 SELECT
     lang_id AS langid,
-    CAST(lower(lang_data_jsonb ->> 'date_format') AS SYS.NCHAR(3)) AS dateformat,
```

Comment:
We need to add the corresponding upgrade changes as well into 2.7.0--2.8.0 upgrade script.